api/src/main/java/org/pmiops/workbench/vwb/usermanager/VwbUserManagerConfig.java

```diff
@@ -35,6 +35,9 @@ public class UserManagerConfig {
   private ApiClient newApiClient(WorkbenchConfig workbenchConfig) {
     ApiClient apiClient = new ApiClient();
     apiClient.setBasePath(workbenchConfig.vwb.userManagerBaseUrl);
+    apiClient.setReadTimeout(30000);
+    apiClient.setConnectTimeout(30000);
+    apiClient.setWriteTimeout(30000);
```

Comment:
why do we need to customize this?

Reply:
I got timeout once when creating the user from VWB so the user creation failed. The default timeout of 10s seems too short.